pmm/pmm2-upgrade-tests.groovy

```diff
@@ -206,7 +206,7 @@ pipeline {
                     sh """
                         set -o errexit
                         set -o xtrace
-                        docker exec pmm-server yum update -y percona-release
+                        docker exec pmm-server yum update -y percona-release || true
```

Comment:
Why do we need `true` here?

Reply:
I have to do it because we have this failing on specific versions, I could see this consistently failing on 2.16.0 with the following 
`Package percona-release-1.0-27.noarch.rpm is not signed`